.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches:
+      - master
+
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false

```

Comment:
Why do you use fail-fast false?
Is not it better to set it to true?

Reply:
If one of the OSs fails in the matrix, but the other doesn't, that probably can give more clues as to what could be the issue.

Say, if something compiled on Ubuntu, but not Windows, then it could be some platform specific issue and that information would be lost with a fail fast approach.